src/xia2/Modules/MultiCrystal/ScaleAndMerge.py

```diff
@@ -1024,9 +1042,12 @@ def report(self):
         )
 
     def cluster_analysis(self):
-        mca = self._mca.cluster_analysis()
-        self._cos_angle_clusters = mca.cos_angle_clusters
-        self._cc_clusters = mca.cc_clusters
+        # mca = self._mca.cluster_analysis()
```

Comment:
Just a bit of tidying required here to remove the commented out lines

Reply:
Done :)